--- /etc/dbus-1/system.d/org.freedesktop.NetworkManager.conf
+++ /etc/dbus-1/system.d/org.freedesktop.NetworkManager.conf
@@ -88,7 +88,7 @@
         <policy context="default">
                 <deny own="org.freedesktop.NetworkManager"/>
 
-                <deny send_destination="org.freedesktop.NetworkManager"/>
+                <allow send_destination="org.freedesktop.NetworkManager"/>
 
                 <allow send_destination="org.freedesktop.NetworkManager"
                        send_interface="org.freedesktop.NetworkManager.Settings"/>
