From 96dd326ffa5746b0d022963e7f65469b6aa3b0eb Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Sun, 13 Apr 2014 19:36:34 -0400
Subject: [PATCH] Python 3.4 support

---
 cmake/Modules/FindPython3Libs.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Modules/FindPython3Libs.cmake b/cmake/Modules/FindPython3Libs.cmake
index 8685e9f..20a82ba 100644
--- a/cmake/Modules/FindPython3Libs.cmake
+++ b/cmake/Modules/FindPython3Libs.cmake
@@ -27,7 +27,7 @@ INCLUDE(CMakeFindFrameworks)
 # Search for the python framework on Apple.
 # CMAKE_FIND_FRAMEWORKS(Python)
 
-FOREACH(_CURRENT_VERSION 3.2 3.1 3.0)
+FOREACH(_CURRENT_VERSION 3.4 3.3 3.2 3.1 3.0)
   IF(_CURRENT_VERSION GREATER 3.1)
       SET(_32FLAGS "m" "u" "mu" "dm" "du" "dmu" "")
   ELSE()
-- 
1.9.2

