From 272bad81aba8b5118f8bb1e8f2db9dcd923ab8aa Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Thu, 10 Apr 2014 14:16:48 -0400
Subject: [PATCH] Simplified defaults for Arch Linux

---
 debian/ubuntu-settings.gsettings-override | 59 +------------------------------
 1 file changed, 1 insertion(+), 58 deletions(-)

diff --git a/debian/ubuntu-settings.gsettings-override b/debian/ubuntu-settings.gsettings-override
index b9300ce..a379e1a 100644
--- a/debian/ubuntu-settings.gsettings-override
+++ b/debian/ubuntu-settings.gsettings-override
@@ -1,6 +1,3 @@
-[org.gnome.crypto.pgp]
-keyservers=['hkp://keyserver.ubuntu.com:11371', 'hkp://pool.sks-keyservers.net', 'ldap://keyserver.pgp.com']
-
 [org.gnome.desktop.background]
 show-desktop-icons=true
 picture-uri='file:///usr/share/backgrounds/warty-final-ubuntu.png'
@@ -8,7 +5,7 @@ picture-uri='file:///usr/share/backgrounds/warty-final-ubuntu.png'
 [org.gnome.desktop.interface]
 gtk-theme="Ambiance"
 icon-theme="ubuntu-mono-dark"
-cursor-theme="DMZ-White"
+cursor-theme="Vanilla-DMZ"
 font-name="Ubuntu 11"
 monospace-font-name="Ubuntu Mono 13"
 
@@ -44,27 +41,6 @@ mouse-button-modifier='<Alt>'
 theme="Ambiance"
 titlebar-font='Ubuntu Bold 11'
 
-[org.gnome.eog.ui]
-external-editor="shotwell-viewer.desktop"
-sidebar=false
-
-[org.gnome.Empathy.conversation]
-theme="adium"
-theme-variant="Normal"
-adium-path="/usr/share/adium/message-styles/ubuntu.AdiumMessageStyle"
-
-[org.gnome.Empathy.notifications]
-notifications-focus=true
-
-[org.gnome.Epiphany]
-keyword-search-url='http://www.google.com/search?client=ubuntu&channel=es&q=%s&ie=UTF-8&oe=UTF-8'
-
-[org.gnome.gedit.plugins]
-active-plugins = ['docinfo', 'modelines', 'filebrowser', 'spell', 'time', 'zeitgeistplugin']
-
-[org.gnome.login-screen]
-logo='/lib/plymouth/ubuntu_logo.png'
-
 [org.gnome.mutter.keybindings]
 toggle-tiled-left = ['<Primary><Super>Left','<Super>Left']
 toggle-tiled-right = ['<Primary><Super>Right','<Super>Right']
@@ -78,42 +54,9 @@ volumes-visible=false
 sort-directories-first=true
 enable-interactive-search=true
 
-[org.gnome.rhythmbox.encoding-settings]
-media-type-presets={'audio/x-vorbis':'Ubuntu', 'audio/mpeg':'Ubuntu'}
-
-[org.gnome.rhythmbox.rhythmdb]
-monitor-library=true
-
-[org.gnome.rhythmbox.plugins]
-active-plugins=['artsearch', 'audiocd','audioscrobbler','cd-recorder','daap','dbus-media-server','generic-player','ipod','iradio','mmkeys','mpris','mtpdevice','notification','power-manager']
-
-[org.gnome.settings-daemon.peripherals.touchpad]
-disable-while-typing = true
-tap-to-click = true
-
-[org.gnome.settings-daemon.plugins.power]
-button-power = 'interactive'
-button-sleep = 'suspend'
-critical-battery-action = 'suspend'
-
 [org.gnome.settings-daemon.plugins.xsettings]
 antialiasing = 'rgba'
 hinting = 'slight'
 
-[org.gnome.settings-daemon.plugins.print-notifications]
-active = false
-
-[org.gnome.settings-daemon.plugins.updates]
-active = false
-
-[org.gnome.settings-daemon.plugins.background]
-active = false
-
-[org.gnome.shell]
-favorite-apps=[ 'ubiquity-gtkui.desktop', 'firefox.desktop', 'evolution.desktop', 'empathy.desktop', 'rhythmbox.desktop', 'shotwell.desktop', 'libreoffice-writer.desktop', 'nautilus.desktop', 'yelp.desktop' ]
-
 [org.gnome.sushi]
 client-decoration = false
-
-[org.gnome.Totem]
-active-plugins = ['skipto','chapters','screenshot','media_player_keys','screensaver','movie-properties','save-file','youtube','totem-zeitgeist-dp-plugin']
-- 
1.9.1

