From fbe13de974247b823b4b27f07ffe1976cd4fa79c Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Thu, 10 Apr 2014 14:16:48 -0400
Subject: [PATCH] Simplified defaults for Arch Linux

---
 debian/ubuntu-settings.gsettings-override | 38 +------------------------------
 1 file changed, 1 insertion(+), 37 deletions(-)

diff --git a/debian/ubuntu-settings.gsettings-override b/debian/ubuntu-settings.gsettings-override
index e56d316..7e07340 100644
--- a/debian/ubuntu-settings.gsettings-override
+++ b/debian/ubuntu-settings.gsettings-override
@@ -1,6 +1,3 @@
-[org.gnome.crypto.pgp]
-keyservers=['hkp://keyserver.ubuntu.com:11371', 'hkp://pool.sks-keyservers.net', 'ldap://keyserver.pgp.com']
-
 [org.gnome.desktop.background]
 show-desktop-icons=true
 picture-uri='file:///usr/share/backgrounds/warty-final-ubuntu.png'
@@ -8,7 +5,7 @@ picture-uri='file:///usr/share/backgrounds/warty-final-ubuntu.png'
 [org.gnome.desktop.interface]
 gtk-theme="Ambiance"
 icon-theme="ubuntu-mono-dark"
-cursor-theme="DMZ-White"
+cursor-theme="Vanilla-DMZ"
 font-name="Ubuntu 11"
 monospace-font-name="Ubuntu Mono 13"
 
@@ -49,23 +46,6 @@ titlebar-font='Ubuntu Bold 11'
 action-middle-click-titlebar='lower'
 titlebar-uses-system-font=false
 
-[org.gnome.eog.ui]
-sidebar=false
-
-[org.gnome.Empathy.conversation]
-theme="adium"
-theme-variant="Normal"
-adium-path="/usr/share/adium/message-styles/ubuntu.AdiumMessageStyle"
-
-[org.gnome.Empathy.notifications]
-notifications-focus=true
-
-[org.gnome.Epiphany]
-keyword-search-url='https://www.google.com/search?client=ubuntu&channel=es&q=%s'
-
-[org.gnome.login-screen]
-logo='/lib/plymouth/ubuntu_logo.png'
-
 [org.gnome.mutter.keybindings]
 toggle-tiled-left = ['<Primary><Super>Left','<Super>Left']
 toggle-tiled-right = ['<Primary><Super>Right','<Super>Right']
@@ -83,15 +63,6 @@ sort-directories-first=true
 enable-interactive-search=true
 open-folder-on-dnd-hover=false
 
-[org.gnome.rhythmbox.encoding-settings]
-media-type-presets={'audio/x-vorbis':'Ubuntu', 'audio/mpeg':'Ubuntu'}
-
-[org.gnome.rhythmbox.rhythmdb]
-monitor-library=true
-
-[org.gnome.rhythmbox.plugins]
-active-plugins=['artsearch', 'audiocd','audioscrobbler','cd-recorder','daap','dbus-media-server','generic-player','ipod','iradio','mmkeys','mpris','mtpdevice','notification','power-manager']
-
 [org.gnome.settings-daemon.plugins.power]
 button-power = 'interactive'
 button-sleep = 'suspend'
@@ -112,16 +83,9 @@ active = false
 [org.gnome.settings-daemon.plugins.background]
 active = false
 
-[org.gnome.shell]
-favorite-apps=[ 'ubiquity.desktop', 'firefox.desktop', 'evolution.desktop', 'rhythmbox.desktop', 'org.gnome.Photos.desktop', 'org.gnome.Nautilus.desktop', 'org.gnome.Software.desktop', 'yelp.desktop' ]
-
 [org.gnome.sushi]
 client-decoration = false
 
-[org.gnome.software]
-first-run=false
-official-sources=['ubuntu-*']
-
 [org.gtk.Settings.FileChooser]
 sort-directories-first = true
 
-- 
2.10.1

