--- a/UnityTweakTool/section/spaghetti/gsettings.py
+++ b/UnityTweakTool/section/spaghetti/gsettings.py
@@ -39,6 +39,9 @@
     elif schema in Gio.Settings.list_schemas():
         pass
     else:
+        # We can't show GUI stuff on a headless build server
+        if 'MAKEPKG' in os.environ and os.environ['MAKEPKG'] == "true":
+          sys.exit()
         print("Error: schema %s not installed" % schema)
         builder=Gtk.Builder()
         builder.set_translation_domain('unity-tweak-tool')
