diff -Nru libpam-freerdp-1.0.1.orig/configure.ac libpam-freerdp-1.0.1/configure.ac
--- libpam-freerdp-1.0.1.orig/configure.ac	2012-09-27 16:11:31.256439035 -0400
+++ libpam-freerdp-1.0.1/configure.ac	2012-09-27 16:12:09.282899693 -0400
@@ -49,16 +49,6 @@
 AC_SUBST(PAMMODULEDIR)
 
 ###########################
-# Google Test Dependencies
-###########################
-
-m4_include([m4/gtest.m4])
-CHECK_GTEST
-if test "x$have_gtest" != "xyes"; then
-   AC_MSG_ERROR([tests were requested but gtest is not installed.])
-fi
-
-###########################
 # gcov coverage reporting
 ###########################
 
@@ -76,6 +66,5 @@
 AC_CONFIG_FILES([
   Makefile
   src/Makefile
-  tests/Makefile
 ])
 AC_OUTPUT
diff -Nru libpam-freerdp-1.0.1.orig/Makefile.am libpam-freerdp-1.0.1/Makefile.am
--- libpam-freerdp-1.0.1.orig/Makefile.am	2012-09-27 16:11:31.253105720 -0400
+++ libpam-freerdp-1.0.1/Makefile.am	2012-09-27 16:12:02.796268169 -0400
@@ -1,7 +1,6 @@
 
 SUBDIRS = \
-	src \
-	tests
+	src
 
 DISTCHECK_CONFIGURE_FLAGS = --enable-localinstall
 
