diff -Nru remote-login-service-1.0.0.orig/src/citrix-server.c remote-login-service-1.0.0/src/citrix-server.c
--- remote-login-service-1.0.0.orig/src/citrix-server.c	2012-09-27 15:30:13.279863387 -0400
+++ remote-login-service-1.0.0/src/citrix-server.c	2012-09-27 15:35:40.891421895 -0400
@@ -86,9 +86,12 @@
 {
 	CitrixServer * server = CITRIX_SERVER(object);
 
-	g_clear_pointer(&server->username, g_free);
-	g_clear_pointer(&server->password, password_clear);
-	g_clear_pointer(&server->domain, g_free);
+	g_free(server->username);
+	server->username = NULL;
+	password_clear(server->password);
+	server->password = NULL;
+	g_free(server->domain);
+	server->domain = NULL;
 
 	G_OBJECT_CLASS (citrix_server_parent_class)->finalize (object);
 	return;
diff -Nru remote-login-service-1.0.0.orig/src/rdp-server.c remote-login-service-1.0.0/src/rdp-server.c
--- remote-login-service-1.0.0.orig/src/rdp-server.c	2012-09-27 15:30:13.279863387 -0400
+++ remote-login-service-1.0.0/src/rdp-server.c	2012-09-27 15:36:56.204347223 -0400
@@ -86,9 +86,12 @@
 {
 	RdpServer * server = RDP_SERVER(object);
 
-	g_clear_pointer(&server->username, g_free);
-	g_clear_pointer(&server->password, password_clear);
-	g_clear_pointer(&server->domain, g_free);
+	g_free(server->username);
+	server->username = NULL;
+	password_clear(server->password);
+	server->password = NULL;
+	g_free(server->domain);
+	server->domain = NULL;
 
 	G_OBJECT_CLASS (rdp_server_parent_class)->finalize (object);
 	return;
diff -Nru remote-login-service-1.0.0.orig/src/uccs-server.c remote-login-service-1.0.0/src/uccs-server.c
--- remote-login-service-1.0.0.orig/src/uccs-server.c	2012-09-27 15:30:13.279863387 -0400
+++ remote-login-service-1.0.0/src/uccs-server.c	2012-09-27 15:37:52.017378193 -0400
@@ -396,7 +396,8 @@
 {
 	g_return_val_if_fail(IS_UCCS_SERVER(server), NULL);
 
-	g_clear_pointer(&server->exec, g_free);
+	g_free(server->exec);
+	server->exec = NULL;
 
 	if (exec != NULL) {
 		server->exec = g_find_program_in_path(exec);
@@ -706,8 +707,10 @@
 		clear_hash(server);
 		clear_json(server);
 
-		g_clear_pointer(&server->username, g_free);
-		g_clear_pointer(&server->password, g_free);
+		g_free(server->username);
+		server->username = NULL;
+		g_free(server->password);
+		server->password = NULL;
 
 		server->username = g_strdup(username);
 		server->password = g_strdup(password);
